packages/commonjs/src/transform.js

```diff
@@ -60,12 +61,56 @@ export function hasCjsKeywords(code, ignoreGlobal) {
   return firstpass.test(code);
 }
 
+function isTrueNode(node) {
+  if (!node) return false;
+
+  switch (node.type) {
+    case 'Literal':
+      return node.value;
```

Comment:
Shouldn't it be `return !!node.value`?

Reply:
It would not make a difference but we can do that.

Reply:
Yeah just to be consistent so you can can always expect a boolean, later you might use this function for other cases